modules/space/rendering/renderableorbitalkepler.cpp

```diff
@@ -296,11 +296,11 @@ void RenderableOrbitalKepler::render(const RenderData& data, RendererTasks&) {
     //glDepthMask(false);
     //glBlendFunc(GL_SRC_ALPHA, GL_ONE)
 
+    GLint* _si = &_startIndex[0];
+    GLint* _ss = &_segmentSize[0];
```

Comment:
More elegant of writing this:
```suggestion
    GLint* _si = _startIndex.data();
    GLint* _ss = _segmentSize.data();
```